integrations/gcp/gcp_core/overrides.py

```diff
@@ -17,7 +17,25 @@ class GCPCloudResourceConfig(ResourceConfig):
     selector: GCPCloudResourceSelector
 
 
+class GCPResourceSelector(Selector):
+    preserve_api_response_case_style: bool | None = Field(
+        default=None,
+        alias="preserveApiResponseCaseStyle",
+        description=(
+            "Controls whether to preserve the google cloud API's original field format instead of using protobuf's default snake case. "
```

Comment:
```suggestion
            "Controls whether to preserve the Google Cloud API's original field format instead of using protobuf's default snake case. "
```